homeassistant/components/lametric/notify.py

```diff
@@ -35,28 +34,39 @@ async def async_get_service(
     coordinator: LaMetricDataUpdateCoordinator = hass.data[DOMAIN][
         discovery_info["entry_id"]
     ]
-    return LaMetricNotificationService(coordinator.lametric)
+    return LaMetricNotificationService(coordinator)
 
 
 class LaMetricNotificationService(BaseNotificationService):
     """Implement the notification service for LaMetric."""
 
-    def __init__(self, lametric: LaMetricDevice) -> None:
+    def __init__(self, coordinator: LaMetricDataUpdateCoordinator) -> None:
         """Initialize the service."""
-        self.lametric = lametric
+        self.coordinator = coordinator
 
     async def async_send_message(self, message: str = "", **kwargs: Any) -> None:
         """Send a message to a LaMetric device."""
         if not (data := kwargs.get(ATTR_DATA)):
             data = {}
 
+        priority = NotificationPriority(data.get(CONF_PRIORITY, "info"))
+        await self.coordinator.async_refresh()
```

Comment:
We could even update the coordinator _only_ when a non-critical message is sent; may cut down on how frequently this needs to be a called.